JobConfig/cosmic/geom_cosmic.txt

```diff
@@ -8,7 +8,7 @@ double world.minimalMargin.xmax = 500000;
 double world.minimalMargin.zmin = 500000;
 double world.minimalMargin.zmax = 500000;
 
-// string dirt.beamline.berm.material = "G4_AIR";
+string dirt.beamline.berm.material = "G4_AIR";
```

Comment:
I don't understand this change, and the same one in geom_cosmic_current. The GitHub history shows this line being commented out in September by Michael, it is also (correctly) commented in PR99, how is it re-appearing?